takari-lifecycle-plugin/src/main/java/io/takari/maven/plugins/compile/jdt/CompilerJdt.java

```diff
@@ -445,6 +449,13 @@ public void addGeneratedSource(Output<File> generatedSource) {
       sources.put(generatedSource.getResource(), generatedSource);
       processedQueue.add(generatedSource.getResource());
     }
+
+    @Override
+    public void onAnnotationProcessing() {
```

Comment:
this method name is what through me.   Perhaps   "enqueAnnotatedFilesForProcessing".    Assuming it's different between strategies at this point.